tests/s3/test_object_workflow_operations.py

```diff
@@ -727,3 +765,184 @@ def test_delete_objects_and_mention_1000_objects_7657(self):
             cfg_7657["del_obj_cnt"])
         self.buckets_list.append(self.bucket_name)
         self.log.info("ENDED: Delete objects and mention 1000 objects.")
+
+    @pytest.mark.parallel
+    @pytest.mark.s3_ops
+    @pytest.mark.s3_object_ops
+    @pytest.mark.tags("TEST-45504")
+    def test_45504(self):
+        """Test GET operation for object with key name consisting of 1024 bytes"""
+        self.log.info(
+            "STARTED: Test GET operation for object with key name consisting "
+            "of 1024 bytes")
+        obj_key_name = "".join(
+            random.choices(
```

Comment:
Use `commons.utils.system_utils.random_string_generator` defined in system_utils.py.

Reply:
Used commons.utils.system_utils.random_string_generator defined in system_utils.py . Done